index.html

```diff
@@ -1,20 +1,41 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Game</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-  <body>
-    <div id="main">
-    <div id="menu">
-      <button id="start">Start game</button><br>
-      <button id="buttonDisc">Game discription</button>
-      <span id="disc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consequuntur similique voluptas, sunt quidem eveniet ad nemo. Incidunt doloremque, dolorem molestias iusto deserunt sapiente et, dolorum perspiciatis laudantium ad. Facilis, eos.</span>
-    </div>
-      <!-- Canvas should have width and height inline or in external css-->
-      <canvas id="canvas" width='566px' height='575px'>Sorry canvas is not supported</canvas>
-    </div>
-    <script src="js/main.js"></script>
-  </body>
+    <head>
+        <meta charset="utf-8"/>
+        <title>
+            Game
+        </title>
+        <link href="css/style.css" rel="stylesheet"/>
+    </head>
+    <body>
+        <div id="main">
+
+            <button id="mute">
+            </button>
+            <div id="menu">
+                <img class="img-menu-girl" src="css/img-menu-girl.gif">
+                <button id="start">
+                </button>
```

Comment:
the same with line break here, as well. Also, for all the same cases in this page.